cpp/include/raft/spatial/knn/detail/processing.hpp

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c)2020, NVIDIA CORPORATION.
```

Comment:
We should update all the years here to `2020-2021`

Reply:
Updated here: https://github.com/rapidsai/raft/pull/126/commits/fe5a378c93b3e1a719b10ec9f72d6a5528e46a07